templates/rss2.tpl

```diff
@@ -26,7 +26,7 @@
 
 		{* optional elements *}
 		{if $context->getPrimaryLocale()}
-			<language>{$context->getPrimaryLocale()|replace:'_':'-'|strip|escape:"html"}</language>
+			<language>{$language|escape}</language>
```

Comment:
As the `$language` doesn't depend on the `$context->getPrimaryLocale()` (which I guess should be always available anyway), then the `{if}` can be removed.